services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/BannerComponents.java

```diff
@@ -250,6 +250,15 @@ public static Builder builder() {
   @Nullable
   public abstract Boolean active();
 
+  /**
+   * A property shows which of the lane's {@link BannerComponents#directions()} is applicable
+   * to the current route, when there is more than one.
```

Comment:
Could you paste the full description from the Directions API docs, including the given example?